src/pybind/mgr/dashboard/frontend/src/app/shared/components/sparkline/sparkline.component.ts

```diff
@@ -48,7 +49,12 @@ export class SparklineComponent implements OnInit, OnChanges {
       enabled: false,
       mode: 'index',
       intersect: false,
-      custom: undefined
+      custom: undefined,
+      callbacks: {
+        label: (tooltipItem) => {
+          return this.dimlessBinaryPipe.transform(tooltipItem.yLabel);
```

Comment:
How about making about using an function given by input here, with this as default if not set? We not always want to show numbers in bytes f.e. under monitoring `open sessions` which comes from `num_sessions` which does not sound like something that shows bytes.

Reply:
Done.
I was not able to call the transform method, if the method was passed via `inputs`, so I now have a `isBinary` as an `input` and will only apply the binary transformation if it is true.